cli/src/templates/packages/react-navigation/navigation/tab-navigator.tsx.ejs

```diff
@@ -37,7 +41,8 @@ export default function TabLayout({ navigation }) {
                 />
               )}
             </Pressable>
-          ),
+            )
```

Comment:
I know Dan said above to remove the formatting changes in the ejs files, but while you're at it, we might as well fix the indentation here ;)